spec/like_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+RSpec.describe Like, type: :model do
+  describe 'Tests for Like model' do
+    usr = User.create(name: 'Ram', photo: 'https://unsplash.com/photos/6PP8DrhY0aA', bio: 'Hindu god.')
+    post = Post.create(author: usr, title: 'Intro', text: 'Welcome to this blog!')
+
+    subject { Like.new }
+    before { subject.save }
+
```

Comment:
Please add a test for `#update_likes_counter` in Like model specs as any custom methods in the models should have corresponding unit tests.